content/en/docs/tasks/observability/distributed-tracing/opentelemetry/index.md

```diff
@@ -1,20 +1,24 @@
 ---
 title: OpenTelemetry
-description: Learn how to configure the proxies to send OpenTelemetry traces to a Collector.
-weight: 10
+description: Learn how to configure the proxies to send traces in OpenTelemetry format.
+weight: 5
 keywords: [telemetry,tracing,opentelemetry,span,port-forwarding]
 aliases:
  - /docs/tasks/telemetry/distributed-tracing/opentelemetry/
+ - /docs/tasks/observability/distributed-tracing/lightstep/
+ - /latest/docs/tasks/observability/distributed-tracing/lightstep/
 owner: istio/wg-policies-and-telemetry-maintainers
 test: yes
 ---
 
+[OpenTelemetry](https://opentelemetry.io/) (OTel) is a vendor-neutral, open source observability framework for instrumenting, generating, collecting, and exporting telemetry data. [OpenTelemetry Protocol](https://opentelemetry.io/docs/specs/otlp/) (OTLP) traces can be sent to [Jaeger](/docs/tasks/observability/distributed-tracing/jaeger/), as well as many commercial services.
+
+To learn how Istio handles tracing, visit this task's [overview](../overview/).
+
 After completing this task, you will understand how to have your application participate in tracing with [OpenTelemetry](https://www.opentelemetry.io/), regardless of the language, framework, or platform you use to build your application.
 
 This task uses the [Bookinfo](/docs/examples/bookinfo/) sample as the example application and the
-[OpenTelemetry Collector](https://opentelemetry.io/docs/collector/) as the receiver of traces.
-
-To learn how Istio handles tracing, visit this task's [overview](../overview/).
+[OpenTelemetry Collector](https://opentelemetry.io/docs/collector/) as the receiver of traces. To see an example of how to send traces directly to an OTLP compatible backend, see the [Jaeger task](/docs/tasks/observability/distributed-tracing/jaeger/).
```

Comment:
Giving this, would we be OK if vendors add their own "tasks", demonstrating how to send OTLP traces to their platform?

Reply:
I removed the Lightstep page thinking that this wouldn't be necessary.  I started off thinking that having a link on the OpenTelemetry page to the vendors that support OTLP would be OK, but ended up removing them too.  So, "I'm not sure", is the current answer, but I think this could be done tastefully.

Reply:
Sounds good! Something we can think/do in a follow up anyway.